test/codeu/chat/server/storage/StorageTest.java

```diff
@@ -49,15 +45,71 @@ public void setup(){
 		}
 	}
 
+	//Resets fields for next tests
+	private void reset() throws SQLException{
```

Comment:
2 Tests should ideally not share any state amongst them. Each test should be isolated and initialize things cleanly from scratch.

You can achieve this by creating local objects within your `test*()` method and test only those. This way each test is isolated and you get implicit cleanup.